### PR TITLE
#23-ConnectEverythingTogether-IrianniMunoz-Co-authored-by: ChristinaMifsud <102205880+christina-mifsud@users.nor…

### PR DESCRIPTION
**Pull request by munozirianni1988**
…eply.github.com>

## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Route, Routes } from "react-router-dom";
-import ModalTestingPage from "./pages/ModalTestingPage";
+//import ModalTestingPage from "./pages/ModalTestingPage";
```
Comment (mferryRV) [inline client/src/App.js]: Rather than commenting it out, go ahead and delete the code. You can also delete the testing page itself. All the code is stored on GitHub so we can look back at it if we need it later.

File: client/src/components/VideoCard.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import React from "react";
 
 function VideoCard({ url }){
-
+    console.log({ url });
     const urlId = url.includes("v=") ? url.split("v=")[1]?.substring(0, 11) : "";
+    console.log({ urlId });
```
Comment (mferryRV) [inline client/src/components/VideoCard.js]: It's always good practice to remove console logs before we push to production

File: client/src/pages/CityIcon.js [hunk 3]
```diff
@@ -20,7 +21,7 @@ const CityIcon = ({ cityName, onClick }) => {
     <div style={cityStyle}>
       <svg onClick={onClick}>
         <circle cx="10" cy="10" r="10" />
-      </svg>
+      </svg>;
```
Comment (mferryRV) [inline client/src/pages/CityIcon.js:24]: Are these changes necessary?

File: client/src/pages/Home.js [hunk 4]
```diff
@@ -25,13 +32,12 @@ export function Home() {
 			});
 	}, []);
 
-	const handleCityClick = (cityName) => {
-		alert(`${cityName} clicked!`);
-	};
+	// const handleCityClick = (cityName) => {
+	// 	alert(`${cityName} clicked!`);
+	// };
```
Comment (mferryRV) [inline client/src/pages/Home.js]: Go ahead and delete this if it's not used

File: client/src/pages/Home.js [hunk 5]
```diff
@@ -65,7 +73,7 @@ export function Home() {
 				<Link to="/DataFetchingTest">Fetch Data from backend</Link>
 				<br></br>
 				<Link to="/about/this/site">About</Link>
-				<Link to="/modaltesting">Modal Testing</Link>
+				{/* <Link to="/modaltesting">Modal Testing</Link> */}
```
Comment (mferryRV) [inline client/src/pages/Home.js]: Delete

File: client/src/pages/Home.js [hunk 6]
```diff
@@ -51,12 +57,14 @@ export function Home() {
...
+						<div key={dataItem.id}>
 						<CityIcon
 							cityName={dataItem.location}
-							onClick={() => handleCityClick(dataItem.location)}
-							key={dataItem.id}
+							onClick={() => handleShow(dataItem)}
 						/>
+						</div>
 					))}
+					<ModalVideo show={show} handleClose={handleClose} title={selectedIcon ? selectedIcon.title:""} url={selectedIcon ?selectedIcon.url: ""} location={selectedIcon ? selectedIcon.location:""} />
```
Comment (mferryRV) [inline client/src/pages/Home.js]: Normally linting will take care of this, but I recommend moving these onto separate lines for readability, e.g.
```jsx
<MyComponent
  show={show}
  otherProp={otherProp} 
  propWithLogic={condition ? true : false} />
```

File: client/src/pages/ModalTestingPage.js [hunk 7]
```diff
@@ -1,27 +1,27 @@
-import React, { useState } from "react";
-import Button from "react-bootstrap/Button";
-import ModalVideo from "../components/ModalVideo";
+// import React, { useState } from "react";
```
Comment (mferryRV) [inline client/src/pages/ModalTestingPage.js]: Delete

**Review by mferryRV — COMMENTED** [review]
Nice work! Please clean up unused code and change the PR name to something clearer, e.g. "Connect Everything Together"

File: client/src/pages/CityIcon.js (hunk 3 above)
Comment (munozirianni1988) [reply client/src/pages/CityIcon.js:24]: Sorry Matthew, I dont understand your question. What changes are you referring to?

**Review by mferryRV — APPROVED** [review]
Nice, much cleaner this way 👍
